api/src/main/java/module-info.java

```diff
@@ -45,7 +45,6 @@
 import jakarta.data.restrict.Restriction;
 
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionStage;
```

Comment:
I think you were only able to remove this because the Javadoc mistakenly has `{@code CompletionStage}` rather than `{@link CompletionStage}`

Reply:
returned the import.